action.yml

```diff
@@ -36,12 +36,19 @@ inputs:
   azure-account-key:
     description: 'Azure Account Name'
     required: false
+  additional-plugins:
+    description: 'List of additional python plugins to use for site creation'
+    required: false
+  skip-publish:
```

Comment:
What is the benefit of this skip option? 🤔 Could we use the other values blank and it would also work?

Reply:
That's true, but I like to have it explicit.